anago

```diff
@@ -200,6 +200,17 @@ check_acls () {
 }
 
 ###############################################################################
+# Ensures that all registries that will be used during both mock and --nomock
+# runs allow write access so we don't fall over later
+# @param registries - A space separated list of registries
+#
+ensure_registry_acls () {
+  local registries=($1)
+
```

Comment:
assuming gcr.io, you could maybe try writing to the GCS bucket backing the registry?  https://cloud.google.com/container-registry/docs/access-control

not sure if that would break anything.